services/wfm/src/main/java/org/openkilda/wfm/topology/utils/KibanaLogManager.java

```diff
@@ -0,0 +1,135 @@
+import org.openkilda.messaging.info.discovery.NetworkDumpSwitchData;
+import org.openkilda.messaging.info.event.IslChangeType;
+import org.openkilda.messaging.info.event.SwitchState;
+import org.openkilda.messaging.model.SwitchId;
+
+import com.google.common.collect.ImmutableMap;
+import org.apache.log4j.Level;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+
+public class KibanaLogManager {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KibanaLogManager.class);
+    /**
+     * The Kibana dashboard tag.
+     */
+    private static final String TAG = "KIBANA_DASHBOARD_TAG";
+
+    public void vfmSync(NetworkDumpSwitchData switchData) {
+        Map<Object, Object> data = ImmutableMap.of(TAG, "Event/WFM Sync", "switch", switchData);
+        proceed("Switch sync", data, INFO);
+    }
+
+    public void vfmSync(NetworkDumpPortData portData) {
+        Map<Object, Object> data = ImmutableMap.of(TAG, "Event/WFM Sync", "switch", portData);
+        proceed("Port sync", data, INFO);
+    }
+
+    public void switchDiscovery(SwitchId switchId, SwitchState state) {
+        Map<Object, Object> data = ImmutableMap.of(TAG, "DISCO: Switch Event", "switch", switchId, "state", state);
+        proceed("Switch event", data, INFO);
+    }
+
+    public void switchEvent(SwitchState state) {
+        Map<Object, Object> data = ImmutableMap.of(TAG, "SWITCH Event", "state", state);
+        proceed("Ignoring state", data, WARN);
+    }
+
+    /**
+     *  Port discovery log.
+     *
+     * @param switchId a switch id.
+     * @param portId a port id.
+     * @param state a port state.
+     */
+    public void portDiscovery(SwitchId switchId, int portId, String state) {
+        Map<Object, Object> data =
+                ImmutableMap.of(TAG, "DISCO: Port Event", "switch", switchId, "port", portId, "state", state);
+        proceed("Port discovery", data, INFO);
+    }
+
+    public void portEvent(String state) {
+        Map<Object, Object> data = ImmutableMap.of(TAG, "PORT Event", "state", state);
+        proceed("Ignoring state", data, WARN);
+    }
+
+    /**
+     * ISL Discovery log.
+     *
+     * @param message a message.
+     * @param srcSwitch a source switch id.
+     * @param srcPort a source port id.
+     * @param dstPort a destination port id.
+     * @param state an ISL state.
+     */
+    public void islDiscovery(String message, SwitchId srcSwitch, Integer srcPort, Integer dstPort,
+                             IslChangeType state) {
+        Map<Object, Object> data = ImmutableMap.of(TAG, "DISCO: ISL Event",
+                "srcSwitch", srcSwitch, "srcPort", srcPort, "dstPort", dstPort, "state", state);
+        proceed(message, data, INFO);
+    }
+
+    public void islEvent(IslChangeType state) {
```

Comment:
... I am dropping review here - where is the detail about ISL? How this message can be used by person who read it?